main.go

```diff
@@ -265,17 +282,17 @@ func mineCmd(cmd *cli.Cmd) {
 		DB := ctx.Value(tellorCommon.DataProxyKey).(db.DataServerProxy)
 		v, err := DB.Get(db.DisputeStatusKey)
 		if err != nil {
-			fmt.Println("ignoring --- could not get dispute status.  Check if staked")
+			level.Warn(logger).Log("msg", "ignoring --- could not get dispute status.  Check if staked")
 		}
 		status, _ := hexutil.DecodeBig(string(v))
 		if status.Cmp(big.NewInt(1)) != 0 {
-			log.Fatalf("Miner is not able to mine with status %v. Stopping all mining immediately", status)
+			level.Error(logger).Log("err", "Miner is not able to mine with status %v. Stopping all mining immediately", status)
```

Comment:
lets be consistent and use `msg` and OTOH in the logger we need to have an even number of argument or use `fmt.Sprintf`

so here you can either have 
`level.Error(logger).Log("msg", fmt.Sprintf("Miner is not able to mine with status %v. Stopping all mining immediately", status))`

or 

`level.Error(logger).Log("msg", "Miner is not able to mine. Stopping all mining immediately","status", status))`
I am a fan of this second version but don't mind either way.

also when you see `log.Fatalf` this means that the should also exit after logging so here you should also add `os.Exit(1)` after the log line.